torch/_inductor/ir.py

```diff
@@ -3690,8 +3701,42 @@ def find_device(tensor_args, example_output):
             return devices[0]
         return None
 
+    def export_extern_kernel_node(self):
```

Comment:
There should be a design comment somewhere describing the overall design of how the proxy executor works (think the high level design doc we hashed through before you started this). Then, this function should cross-reference with that design doc. If you like, you can make the gdoc public and link it here.

Reply:
Done.